### DISCUSSION TITLE
Registry could show available targets for a module!

### DISCUSSION THREAD

**Discussion by Attempt3035** [bazelbuild/bazel-central-registry#1301] [Ideas]
Hi all!
I've been thinking it would be really handy if modules displayed on registry.bazel.build could also display the targets that they have (which could be grabbed from the verified targets in the presubmit?)
Even better, is there any way we could provide module creators a way to submit a short readme or extra description in the metadata.json that could provide any additional instructions?
Any ideas or thoughts?

**Comment by meteorcloudy** [comment]
Currently registry.bazel.build shows the home page of each module if provided. For those modules, I think it's better to keep the readme and user doc in their own repo so it's always up-to-date.

For other modules whose project doesn't natively support Bazel/Bzlmod, this could be helpful. But I don't feel very strongly about it.
